### PR TITLE
Simplify linting

### PR DESCRIPTION
**Pull request by m-murasovs**
I tried to separate linting for Markdown and JS but turned out to be pretty annoying to specify which one to lint each time.

Joining all the lint commands into one

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,12 +4,10 @@
...
   "scripts": {
-    "build": "npm run lint:md && npm run lint:code && node ./src/build.js",
+    "build": "npm run lint:fix && node ./src/build.js",
     "test": "npm run test",
-    "lint:md": "./node_modules/.bin/markdownlint '**/*.md'",
-    "lint:md:fix": "./node_modules/.bin/markdownlint '**/*.md' --fix",
-    "lint:code": "./node_modules/.bin/eslint . && eslint --ext js,md --fix .",
-    "lint:code:fix": "./node_modules/.bin/eslint . --ext .js,.jsx --fix && eslint --ext js,md --fix ."
+    "lint": "./node_modules/.bin/markdownlint '**/*.md' && ./node_modules/.bin/eslint . && eslint --ext js,md",
+    "lint:fix": "./node_modules/.bin/markdownlint '**/*.md' --fix && ./node_modules/.bin/eslint . --ext .js,.jsx --fix && eslint --ext js,md --fix ."
```
Comment (mtrunkat) [inline package.json:10]: Wouldn't be better to have all variants? It makes also package.json more readable:
```
"lint:md": "...",
"lint:md:fix": "...",
"lint:code": "...",
"lint:code:fix": "...",
"lint": "npm run lint:md && npm run lint:code",
"lint:fix": "npm run lint:md:fix && npm run lint:code:fix",
```

**Review by mtrunkat — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mtrunkat) [reply package.json:10]: I think it's more readable as commands are shorter

File: package.json [hunk 2]
```diff
@@ -6,9 +6,11 @@
   "scripts": {
     "build": "npm run lint:md && npm run lint:code && node ./src/build.js",
     "test": "npm run test",
+    "lint": "./node_modules/.bin/markdownlint '**/*.md' && ./node_modules/.bin/eslint . && eslint --ext js,md",
```
Comment (mtrunkat) [inline package.json:9]: @m-murasovs , can't you make this simply:

```
"lint": "npm run lint:code && npm run lint:md"
```

and the same for "lint:fix" to get a rid of that duplicate code?

Comment (m-murasovs) [reply package.json:9]: Ah, I'm not that smart!

**Review by mtrunkat — APPROVED** [review]

**Pull request merged**